tests/test_modeling_physical_validation.py

```diff
@@ -0,0 +1,161 @@
+"""
+Tests for modeling
+"""
+
+import os
+import ast
+import astropy
+from numpy import testing as tst
+import numpy as np
+import clmm
+from clmm.constants import Constants as clmmconst
+
+# Read test case
+CODE_PATH = '/'.join(os.path.abspath(__file__).split('/')[:-2])
+VAL_FILES_PATH = '%s/tests/physical_values_nc'%CODE_PATH
```

Comment:
We shouldn't need to add anything to path so relative path work. `pytest` runs from the root package directory.

Reply:
fixed